### ISSUE TITLE
Labor Admin "Approve" and "Deny" Overload Form

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#301]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/301) by May Jue (Bitbucket: [May Jue](https://bitbucket.org/May Jue), ).**

----------------------------------------

Labor Admin gets three options for Overload form applications. Approve, Approve reluctantly, and Deny. Labor admin should only be able to approve the form \(NOT approve reluctantly\) after the SASS and Financial Aid approve the form. Currently, the SASS and FA approval are not required before the labor admin approving the form and after SASS and FA deny the form, the labor admin can still approve the form successfully. This should be fixed so that the labor admin can “approve” the form only after SASS and Financial Aid give approval.  
  
When the Labor admin “deny” an Overload form. The LSF for that overload is passed to LSF pending form when “deny” the overload form should result in that LSF being deleted.

![](https://lh3.googleusercontent.com/NcWuYe2tQis-o6kmTif4ojW5-8vcEHDpEcPIa1A3c96UhLGzhgwdfN8FotkYVlbaBj3qbLs1-kIc0pj0F7KVFU5ISgzVZcELthUt6sxJcCTw6Cp8et3X4tWlxB7bBwD-ABs_i-yW)
‌

For more information:  
[https://docs.google.com/document/d/1pHH0Ld9cKeEiKGsRJYzBBVaXZHvJzxsU6tdJyV5EiEY/edit](https://docs.google.com/document/d/1pHH0Ld9cKeEiKGsRJYzBBVaXZHvJzxsU6tdJyV5EiEY/edit)

‌

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

The second part of the issue about denied forms showing in pending LSF is fixed in another PR.

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

We decided not need to put any restrictions on labor admins. Therefore, I will be closing this issue.
